datalad/cmd.py

```diff
@@ -194,8 +194,8 @@ def _log_out(self, line):
     def _log_err(self, line, expected=False):
         if line and self.log_outputs:
             self.log("stderr| " + line.rstrip('\n'),
-                     level={True: logging.DEBUG,
-                            False: logging.ERROR}[expected])
+                     level={True: 9,
+                            False: 11}[expected])
```

Comment:
Why to log errors not at error level?

Reply:
To avoid shocking users if we missed one, that is actually treated in higher level code. Those logs are important for developers only, I think and we can still have a run at log level 11 to include them without having all the DEBUG output. See #1753 . That's the compromise nobody responded to. ;-)
Despite all that: It's not necessarily an actual error, but just an unexpected output on stderr or an unexpected non-zero return. Even the latter is an error only from the perspective of the command we called - doesn't mean it's an error from datalad's point of view or the user's. The only thing it does mean, is that WE either were not aware that could happen or we failed to properly avoid the ERROR level when we catch that "error" anyway. Either way: It's a hint for datalad developers, nobody else.

Reply:
And what's the relation to custom remotes here?

Reply:
was reviewing on the phone, mixed up the files I guess ;)